fuzzers/klee/fuzzer.py

```diff
@@ -38,7 +38,8 @@ def prepare_build_environment():
     os.environ['CC'] = 'gclang'
     os.environ['CXX'] = 'gclang++'
 
-    os.environ['FUZZER_LIB'] = '/libAFL.a -L/ -lKleeMock'
+    os.environ['FUZZER_LIB'] = '/libAFL.a -L/ -lKleeMock -lpthread'
```

Comment:
I think `-pthread` is set by default in CXXFLAGS and CFLAGS.
Is it actually needed?

Reply:
It was needed to resolve an issue with libpng, though the rest of that fix is missing in this PR.